pkg/resources/allocator.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018 Intel Corp. All Rights Reserved.
```

Comment:
Should it be 2022?

Reply:
Yes, changed.
https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/443/files#diff-f19d6492bf6a94ca5a69d2f87d38a7de8430d4c85bd33b76e0dbb80bb0327326R1